recipes-ni/initscripts-nilrt/files/nisetreboottype

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# The purpose of this script is to set the reboot type of the controller based
+# on the value of ni_boot_mode_input. Reboots to install mode require a cold
+# reboot. All other modes should use the default reboot type
```

Comment:
I don't know if this is a relevant workflow; it seems like the remnants of the obsolete "install mode" workflow. Can you check if SCS or anyone else even sets `/sys/kernel/ni_requested_reboot_type` to `install`? If not, we can drop this whole file.

Reply:
Found [this](https://p4.natinst.com/browser/perforce/MAX/components/remote/softwareinstall/trunk/21.0/source/SoftwareActionShared/Imaging.cpp?revision_at=51501451#line_388)... so keep this around?

Reply:
I... don't know what that's supposed to mean? Codepath run when `reboot_type` is set to `install`?

Reply:
The codepath runs when ni_boot_mode_input is set to install and [this](https://p4.natinst.com/browser/perforce/MAX/components/remote/softwareinstall/trunk/21.0/source/SoftwareActionShared/Imaging.cpp?revision_at=51501451#line_388) does that.